### PR TITLE
Add usage instructions to AroundTheWorld README

### PR DESCRIPTION
**Pull request by tbconroy**

### CODE REVIEW

File: around_the_world/README.md [hunk 1]
```diff
@@ -22,16 +22,95 @@ gem "around_the_world"
...
+```ruby
+class SomeClass
+  include AroundTheWorld
+
+  def did_something_happened?
+    true
+  end
+
+  around_method :did_something_happened? do
+    things_happened = super()
```
Comment (arettberg) [inline around_the_world/README.md]: these parentheses are hurting my eyes

Comment (tbconroy) [reply around_the_world/README.md]: ☹️ `ruby 2.5.5` throws this without the parens:
```
RuntimeError (implicit argument passing of super from method defined by define_method() is not supported. Specify all arguments explicitly.)
```

Comment (arettberg) [reply around_the_world/README.md]: oh interesting - in that case, let's actually update the docs (here and the rdoc) to instead use the format:
```ruby
around_method :some_method do |*args|
  super *args
end
```

Comment (garside) [reply around_the_world/README.md]: I *think* you can just specify an implicit glob for this syntax?

```
around_method :some_method do |*|
  super(*)
end
```

Comment (arettberg) [reply around_the_world/README.md]: mmm nope that's def a syntax error

Comment (tbconroy) [reply around_the_world/README.md]: Updated the README and the YARD comments

**Pull request merged**